src/main/java/com/softserveinc/ita/dkrutenko/pageobjects/BasePage.java

```diff
@@ -59,9 +59,13 @@ public WebElement waitForElementVisibility(By locator) {
         return  driverWait.until(numberOfElementsToBeMoreThan(locator, amount));
     }
 
+    public WebElement setSearch() {
+        return driver.findElement(searchFieldSelector);
+    }
+
     public void fillSearchField(String text) {
-        driver.findElement(searchFieldSelector).click();
-        driver.findElement(searchFieldSelector).clear();
-        driver.findElement(searchFieldSelector).sendKeys(text);
+        setSearch().click();
```

Comment:
and what you have changed with such implementation? you look at the same element three times and init three times

I mean such implementation:

var element = driver.findElement(searchFieldSelector);

element.click();
element.clear();
element.sendKeys(text);

Reply:
done